presto-hive/src/main/java/com/facebook/presto/hive/HiveColumnHandle.java

```diff
@@ -167,14 +169,14 @@ public boolean equals(Object obj)
     @Override
     public String toString()
     {
-        return toStringHelper(this)
+        ToStringHelper helper = toStringHelper(this)
                 .add("clientId", clientId)
                 .add("name", name)
                 .add("hiveType", hiveType)
                 .add("hiveColumnIndex", hiveColumnIndex)
-                .add("columnType", columnType)
-                .add("comment", comment)
-                .toString();
+                .add("columnType", columnType);
+        comment.ifPresent(comment -> helper.add("comment", comment));
```

Comment:
The other way to handle this is to use `https://google.github.io/guava/releases/19.0/api/docs/com/google/common/base/MoreObjects.ToStringHelper.html#omitNullValues()` and `Optional.orElse(null)`. That way you would not need to break the method call chain.

Reply:
`omitNullValues()` is preferred as that's how we do it elsewhere